### PR TITLE
added fix for https://issues.jboss.org/browse/AF-1733

### PR DESCRIPTION
**Pull request by ghjboss**
It introduces a system property org.uberfire.nio.git.http.sslVerify which controls to turn off/on the sslVerify.
Only if org.uberfire.nio.git.http.sslVerify is set as false the sslVerify will be turned off.

### CODE REVIEW

File: uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/commands/CreateRepository.java [hunk 1]
```diff
@@ -77,6 +78,17 @@ public CreateRepository(final File repoDir,
                     .build();
 
             final org.eclipse.jgit.api.Git git = new org.eclipse.jgit.api.Git(repo);
+            
+            String ssl =  System.getProperty("org.uberfire.nio.git.http.sslVerify");
```
Comment (ederign) [inline uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/commands/CreateRepository.java]: This property should be placed here: https://github.com/kiegroup/appformer/blob/dd03638ed7aba6146e042b7a7062d9ad4630bd3f/uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProviderConfiguration.java#L29

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by ghjboss** [comment]
The use case to disable the sslVerify is that some users run into issues when using https to clone a git project due to the ssl verification. 
Examples;
Caused by: javax.net.ssl.SSLHandshakeException: java.security.cert.CertificateException: No name matching xxx found
The simplest way to work this around is to turn off the http sslVerify.

I've made some changes in the code. The property org.uberfire.nio.git.http.sslVerify is moved to the class JGitFileSystemProviderConfiguration. However, I don't set the property like others in this class in the load(ConfigProperties systemConfig) method.  That will need to create a JGitFileSystemProviderConfiguration instance in the CreateRepository class to retrieve its property.  I simply put the system property there and call System.getProperty(JGitFileSystemProviderConfiguration.GIT_HTTP_SSLVERIFY) to read the property.

I"m not sure how a test case could be created here.  If somebody could help I would appreciate.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by ederign** [comment]
ok to test

**Comment by ederign** [comment]
@ghjboss I'll work on the tests and ping you back

**Comment by kiereleaseuser2** [comment]
Build finished. 5027 tests run, 11 skipped, 0 failed.

**Comment by ederign** [comment]
A updated version: https://github.com/kiegroup/appformer/pull/562/commits

**Pull request closed without merging**